.github/workflows/kakarot_rpc.yml

```diff
@@ -78,19 +79,45 @@ jobs:
         with:
           cache-on-failure: true
       - uses: taiki-e/install-action@cross
+      - name: Build
+        run: |
+          RUSTFLAGS="-C link-arg=-lgcc -Clink-arg=-static-libgcc" \
+            cross build --bin kakarot-rpc --release --target ${{ matrix.target }}
+          mkdir -p ./bin/${{ matrix.platform }}
+          mv target/${{ matrix.target }}/release/kakarot-rpc ./bin/${{ matrix.platform }}/kakarot-rpc
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: build-${{ matrix.platform }}
+          path: ./bin/${{ matrix.platform }}/kakarot-rpc
+
+  build-push-docker:
+    runs-on: ubuntu-latest-16-cores
+    needs: build
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/download-artifact@v4
+        with:
+          name: build-amd64
+          path: ./bin/amd64/kakarot-rpc
+      - uses: actions/download-artifact@v4
+        with:
+          name: build-arm64
+          path: ./bin/arm64/kakarot-rpc
       - name: Log in to Docker
         run: |
           echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io --username ${DOCKER_USERNAME} --password-stdin
       - name: Set up Docker builder
         run: |
-          docker run --privileged --rm tonistiigi/binfmt --install ${{ matrix.platform }}
+          docker run --privileged --rm tonistiigi/binfmt --install amd64
```

Comment:
@greged93 i kept this here but not sure what it does

Reply:
I saw this in Reth so I'm not 100% sure but I think it speeds up the docker buildx command.